### PR TITLE
README fixes for emulation setup

### PR DESCRIPTION
**Pull request by sampsyo**
Hello! This tweaks a few things in the README for the emulation setup instructions.

- Fix Markdown formatting to use code blocks.
- Fix venv instructions to describe activation (instead of duplicating the same command twice).
- Fix a reference to a nonexistent branch in the `clone` command.
- Link to helpful background on venvs.
- Switch to `python3` instead of `python3.6` to be a tad more flexible.
- Remove a `cd` that is invalidated by the previous step—perhaps we need to add one back somewhere?
- English style tweaks.

### CODE REVIEW

**Review by yodada — APPROVED** [review]
Thank you @sampsyo !

**Comment by yodada** [comment]
@sampsyo just added you to the group that can push to master. now you should be able to click the merge button ;)

**Comment by sampsyo** [comment]
Wahoo! Looks like some required Travis job flaked out, so maybe I'll give it a little time before merging manually.

**Comment by yodada** [comment]
yeah that job just disappeared ... hmm ... should we just go ahead and merge this one? we are not changing any code anyways

**Pull request merged**
